concept/thing/attribute.rs

```diff
@@ -267,7 +277,8 @@ where
                 None => self.state = State::Done,
                 Some(Ok(attribute)) => {
                     let attribute_vertex = attribute.vertex();
-                    let independent = self.independent_attribute_types.contains(&attribute.type_());
+                    // let independent = self.independent_attribute_types.contains(&attribute.type_());
+                    let independent = true; // TODO: Turn off filtering, come up with a complete solution later
```

Comment:
Am I right that I can just remove the independence check here?

Reply:
I need to have a deeper think about this one tomrrow/on monday with you, will come back to it